Common/Basic comppenants/CatalogCategories.cs

```diff
@@ -0,0 +1,24 @@
+﻿using OpenQA.Selenium;
+using System.Linq;
+
+namespace Infrastructure
+{
+    public class CatalogCategories : ComppenantBase
+    {
+        public Button Women => ParentElement.FindElements(By.CssSelector("#block_top_menu ul.sf-menu li a.sf-with-ul")).Select(element => new Button(Driver, element)).ToList()[0];
```

Comment:
why take the first one and not select a specific one by name? like you did in dresses and tshirts